include/barrett/bus/can_socket.h

```diff
@@ -58,14 +58,14 @@ class CANSocket : public CommunicationsBus {
 	/** CANSocket() Constructors
 	 */
 	CANSocket();
-	CANSocket(int port) throw(std::runtime_error);
+	CANSocket(int port);
```

Comment:
Can we add `explicit` keyword here to disallow hidden bugs?

Reply:
Done!